src/main/java/io/appform/dropwizard/actors/actor/MessageProperties.java

```diff
@@ -0,0 +1,15 @@
+package io.appform.dropwizard.actors.actor;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class MessageProperties {
```

Comment:
Rename class, remove everything other than `@Data` .. this should be constructed and final .

Reply:
removed annotations and made class final.